### DISCUSSION TITLE
Spark Doris Connector 1.3写入Doris 2.1.1报错

### DISCUSSION THREAD

**Discussion by CookingBoy** [apache/doris#35163] [A - General / Q&A]
错误日志如下：
W20240521 15:20:20.674120 24060 stream_load.cpp:351] append body content failed. errmsg=[INTERNAL_ERROR]cancelled: closed, id=8e47a69136a616de-46fc3d859099eeb6, job_id=-1, txn_id=2204, label=spark_streamload_20240521_152017_d9611311effe4f9198ee66863906e558, elapse(s)=3
W20240521 15:20:20.676141 22447 vtablet_writer.cpp:600] cancel node channel VNodeChannel[1766051-10004], load_id=8e47a69136a616de-46fc3d859099eeb6, txn_id=2204, node=10.74.1.149:5060, error message: [END_OF_FILE]Encountered unqualified data, stop processing
W20240521 15:20:20.677246 22496 fragment_mgr.cpp:409] report error status: [END_OF_FILE]Encountered unqualified data, stop processing to coordinator: TNetworkAddress(hostname=10.74.1.149, port=9021), query id: 8e47a69136a616de-46fc3d859099eeb6, instance id: 8e47a69136a616de-46fc3d859099eeb7
W20240521 15:20:20.677629 22496 stream_load_executor.cpp:113] fragment execute failed, query_id=0000000000000000-0000000000000000, err_msg=[CANCELLED][END_OF_FILE]Encountered unqualified data, stop processing, id=8e47a69136a616de-46fc3d859099eeb6, job_id=-1, txn_id=2204, label=spark_streamload_20240521_152017_d9611311effe4f9198ee66863906e558, elapse(s)=3
W20240521 15:20:20.715088 22448 vtablet_writer.cpp:600] cancel node channel VNodeChannel[1766051-10004], load_id=454c7758e448ec7f-a65ac352a81268b3, txn_id=2203, node=10.74.1.149:5060, error message: [END_OF_FILE]Encountered unqualified data, stop processing
W20240521 15:20:20.730255 24061 stream_load.cpp:351] append body content failed. errmsg=[INTERNAL_ERROR]cancelled: closed, id=454c7758e448ec7f-a65ac352a81268b3, job_id=-1, txn_id=2203, label=spark_streamload_20240521_152017_989f9a2cf1594fdbb968ad1650f0344e, elapse(s)=3
W20240521 15:20:20.731957 22496 fragment_mgr.cpp:409] report error status: [END_OF_FILE]Encountered unqualified data, stop processing to coordinator: TNetworkAddress(hostname=10.74.1.149, port=9021), query id: 454c7758e448ec7f-a65ac352a81268b3, instance id: 454c7758e448ec7f-a65ac352a81268b4
W20240521 15:20:20.732309 22496 stream_load_executor.cpp:113] fragment execute failed, query_id=0000000000000000-0000000000000000, err_msg=[CANCELLED][END_OF_FILE]Encountered unqualified data, stop processing, id=454c7758e448ec7f-a65ac352a81268b3, job_id=-1, txn_id=2203, label=spark_streamload_20240521_152017_989f9a2cf1594fdbb968ad1650f0344e, elapse(s)=3

请问有什么定位的思路？

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题
